### ISSUE TITLE
How to manually delete files created by offline.py --create_mode?

### ISSUE THREAD

**Issue by kristiank** [spraakbanken/karp-backend-v5#4]
How can I manually delete the files `python offline.py --create_mode` created when I don't have the server running?

```bash
curl -XDELETE 'localhost:8083/karp_test'
curl: (7) Failed connect to localhost:8083; Connection refused
```
I get `Index allready exists` error when I try to (re)create the mode:
```bash
python offline.py --create_mode karp test
ConnectionError(<urllib3.connection.HTTPConnection object at 0x15b3050>: Failed to establish a new conneced by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x15b3050>: Failed to establish a nown)
Traceback (most recent call last):
  File "offline.py", line 14, in <module>
    upload.create_mode(mode, suffix)
  File "/home/centos/spraakbanken/karp-backend/src/offline/upload_offline.py", line 269, in create_mode
    raise Exception('Index already exists')
Exception: Index already exists
```

**Comment by kristiank** [comment]
When trying to publish my mode I get the following error message, which seems to indicate that Elasticsearch is not running or properly set up (as Issued here #3).
```bash
python offline.py --publish_mode karp test
Publish karp_test as karp
remove ['{"remove": {"index":"karp_*", "alias":"karp"}}']
add ['{"add" : {"index": "karp_test", "alias": "karp"}}']
remove old aliases
No previous aliased indices, could not do remove any
['{"remove": {"index":"karp_*", "alias":"karp"}}']
Traceback (most recent call last):
  File "offline.py", line 20, in <module>
    upload.publish_group(group, suffix)
  File "/home/centos/spraakbanken/karp-backend/src/offline/upload_offline.py", line 251, in publish_group
    return es.indices.update_aliases('{"actions" : [%s]}' % ','.join(add_actions))
  File "/home/centos/spraakbanken/karp-backend/venv/lib/python2.7/site-packages/elasticsearch/client/utils.py", line 76, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/home/centos/spraakbanken/karp-backend/venv/lib/python2.7/site-packages/elasticsearch/client/indices.py", line 403, in update_aliases
    params=params, body=body)
  File "/home/centos/spraakbanken/karp-backend/venv/lib/python2.7/site-packages/elasticsearch/transport.py", line 314, in perform_request
    status, headers_response, data = connection.perform_request(method, url, params, body, headers=headers, ignore=ignore, timeout=timeout)
  File "/home/centos/spraakbanken/karp-backend/venv/lib/python2.7/site-packages/elasticsearch/connection/http_urllib3.py", line 158, in perform_request
    raise ConnectionError('N/A', str(e), e)
elasticsearch.exceptions.ConnectionError: ConnectionError(<urllib3.connection.HTTPConnection object at 0x2239050>: Failed to establish a new connection: [Errno -2] Name or service not known) caused by: NewConnectionError(<urllib3.connection.HTTPConnection object at 0x2239050>: Failed to establish a new connection: [Errno -2] Name or service not known)
```

**Comment by MalinAhlberg** [comment]
First of all: sorry for the stupid error message!
The problem is, as you have already noticed, not that the index already exists. It is possible to see that in the first error as well, on line 2, but it is not very clear at all...
If there would have been an index that you needed to get rid of, you can do that by communicating directly with ES: `curl -XDELETE localhost:8083/karp_test`. 

I hope, however, that your current problem will be solved by changing the configs as in issue #3 .
